Upstream-Status: Pending

This patch fixes following issue with automake 1.12

| configure.ac:80: error: automatic de-ANSI-fication support has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/03

Index: findutils-4.4.2/configure.ac
===================================================================
--- findutils-4.4.2.orig/configure.ac
+++ findutils-4.4.2/configure.ac
@@ -77,8 +77,6 @@ AC_PROG_CPP
 dnl for gnulib
 gl_EARLY
 
-AM_C_PROTOTYPES
-
 AC_PROG_INSTALL
 AC_PROG_RANLIB
 dnl AC_PROG_LIBTOOL
